addons/mail/static/src/widgets/form_renderer/form_renderer_tests.js

```diff
@@ -49,6 +51,195 @@ QUnit.module('form_renderer_tests.js', {
     },
 });
 
+QUnit.test('[technical] messaging not created', async function (assert) {
+    assert.expect(3);
+
+    this.data['res.partner'].records.push({ display_name: "second partner", id: 12, });
+    await this.createView({
+        data: this.data,
+        hasView: true,
+        messagingBeforeCreationDeferred: makeDeferred(), // block messaging creation
+        waitUntilMessagingCondition: 'none',
+        // View params
+        View: FormView,
+        model: 'res.partner',
+        arch: `
+            <form string="Partners">
+                <sheet>
+                    <field name="name"/>
+                </sheet>
+                <div class="oe_chatter"></div>
+            </form>
+        `,
+        res_id: 12,
+    });
+    assert.containsOnce(
+        document.body,
+        '.o_ChatterContainer',
+        "should display chatter container even when messaging is not created yet"
+    );
+    assert.containsOnce(
+        document.body,
+        '.o_ChatterContainer_noChatter',
+        "chatter container should not display any chatter when messaging not created"
+    );
+    assert.strictEqual(
+        document.querySelector('.o_ChatterContainer').textContent,
+        "Please wait...",
+        "chatter container should display spinner when messaging not yet created"
+    );
+});
+
+QUnit.test('messaging becoming created but not initialized', async function (assert) {
```

Comment:
what makes the first test here "technical" and this one not?

Reply:
Should also be technical IMO. The consideration of messaging not being created is poor but unfortunately necessity considering the current way of patching/extending stuff with JS modules.